src/System.Reflection.DispatchProxy/tests/DispatchProxyTests.cs

```diff
@@ -386,5 +386,94 @@ public static void Invoke_Property_Setter_And_Getter_Invokes_Correct_Methods()
 
             Assert.Null(actualValue);
         }
+
+
+        [Fact]
+        public static void Proxy_Declares_Interface_Properties()
+        {
+            TestType_IPropertyService proxy = DispatchProxy.Create<TestType_IPropertyService, TestDispatchProxy>();
+            PropertyInfo propertyInfo = proxy.GetType().GetTypeInfo().GetDeclaredProperty("ReadWrite");
+            Assert.NotNull(propertyInfo);
+        }
+
```

Comment:
More test. I saw a test that invokes the method for an indexed property (“Item”), but not the equivalent for a non-indexed property (“ReadWrite”) (If it’s there and I just didn’t see it, feel free to ignore.)

The non-indexed property is a pretty important edge case so I figured it deserved the same test coverage.